--- link-grammar-5.7.0/bindings/python/Makefile.am.orig	2023-06-02 19:11:33.020036329 +0200
+++ link-grammar-5.7.0/bindings/python/Makefile.am	2023-06-02 19:11:35.733354963 +0200
@@ -57,7 +57,6 @@ _clinkgrammar_la_SOURCES = $(built_c_sou
 _clinkgrammar_la_CPPFLAGS =       \
    $(SWIG_PYTHON_CPPFLAGS)        \
    $(PYTHON2_CPPFLAGS)            \
-   -I$(top_srcdir)                \
    -I$(top_builddir)
 
 # On Cygwin and MinGW, a DLL with version is named name-major.EXT with no
@@ -75,7 +74,7 @@ _clinkgrammar_la_LDFLAGS =
     -avoid-version \
     $(PYTHON2_LDFLAGS) -module -no-undefined $(PYMODULE_EXT)
 _clinkgrammar_la_LIBADD =                         \
-    $(top_builddir)/link-grammar/liblink-grammar.la $(PYTHON2_LIBS)
+    -llink-grammar $(PYTHON2_LIBS)
 
 
 EXTRA_DIST =         \
